content/articles/Factor-that-affects-the-performance-deep-learning-systems/index.md

```diff
@@ -0,0 +1,114 @@
+### INTRODUCTION
+Deep learning is a type of artificial intelligence that simulates how humans learn specific sorts of information. It's a branch of AI and machine learning. This technology has been used to create many self-driving cars and smart digital devices.
+ 
+When given raw data, a deep learning system chooses which attributes are most relevant on its own. Deep learning is carried out using [artificial neural networks](https://www.simplilearn.com/tutorials/deep-learning-tutorial/multilayer-perceptron) with multiple layers. 
+Deep Neural Networks (DNNs) are networks that conduct difficult operations such as representation and abstraction at each layer to make sense of images, music, and text. 
+Deep learning is the fastest-growing field in machine learning, and an increasing number of firms are using it to create new business models.
+### Table Of Content
+-	[Introduction](#INTRODUCTION)
+-	[Prerequisites](#Prerequisites)
+-	[The Concept of Deep Learning](#The-Concept-of-Deep-Learning)
+-	[Some Determinants That Transform Deep Learning Systems’ Performance](#Some-Determinants-That-Transform-Deep-Learning-Systems-Performance)
+-	[How To Get A Perfect Performance Rate From A Deep Learning System](#How-To-Get-A-Perfect-Performance-Rate-From-A-Deep-Learning-System)
+-	[Diagram demonstrating the different scenarios one can fall into when configuring the learning rate](#Diagram-demonstrating-the-different-scenarios-one-can-fall-into-when-configuring-the-learning-rate-.)
+-	[Conclusion](#CONCULSION)
+-	[Further Reading](#Further-Reading)
+
+### Prerequisites
+To follow along with this tutorial, you need to be familiar with the following;
+-	Programming Languages like Python, Java, C++
+-	Calculus
+-	Probability
+-	Statistics
+-	Linear Algebra
+-	[TensorFlow](https://www.google.com/url?sa=t&source=web&rct=j&url=https://www.tensorflow.org/&ved=2ahUKEwi15PyaroL1AhWmS2wGHZZUBdMQFnoECF4QAQ&sqi=2&usg=AOvVaw0TGZBeXHx2CVPI2FiDZclR) 
```

Comment:
`https://www.google.com/url?sa=t&source=web&rct=j&url=https://www.tensorflow.org/&ved=2ahUKEwi15PyaroL1AhWmS2wGHZZUBdMQFnoECF4QAQ&sqi=2&usg=AOvVaw0TGZBeXHx2CVPI2FiDZclR` -> `https://www.tensorflow.org`

I believe you have copied the URLs directly from a google search. These URLs gets new tokens every week. So, the week after these URLs might not work. So, it's best to use the direct domains.

Reply:
Do the same for all other URLs